lib/templateFactory.js

```diff
@@ -104,18 +108,19 @@ class TemplateFactory extends BaseFactory {
                 });
         }
 
-        const [, namespace, name] = TEMPLATE_NAME_REGEX_WITH_NAMESPACE.exec(fullTemplateName);
+        const [, namespace, name] =
+            TEMPLATE_NAME_REGEX_WITH_NAMESPACE.exec(fullTemplateName);
 
         // Check if template with namespace and name exist, default to using that template
         return super
             .list({
                 params: {
                     namespace,
-                    name
+                    name,
                 },
-                paginate
+                paginate,
             })
-            .then(namespaceExists => {
+            .then((namespaceExists) => {
```

Comment:
the added parens will fail lint

Reply:
What formatter do you guys use?

Reply:
```
node ./node_modules/.bin/eslint --fix .
```

You can run the above code, warnings are fine, but any errors will fail